pkg/importer/dataStream_test.go

```diff
@@ -14,10 +14,10 @@ import (
 	"testing"
 
 	"kubevirt.io/containerized-data-importer/pkg/image"
-	"kubevirt.io/containerized-data-importer/test/framework"
+	"kubevirt.io/containerized-data-importer/tests/framework"
```

Comment:
Should this be like this, or was it just a global search and replace that went wrong?

Reply:
framework/ was moved under tests/ from test/.  Given that we don't have a unified package for the refactored unit tests, this seemed like a reasonable place for it.  I'm open to suggestions though.

Reply:
I've updated the PR description to list the packages / files that were moved.

Reply:
@copejon My pr 349 creates _tests/framewor/_ for common e2e funcs, so we need to coordinate

Reply:
@jeffvance This should be handled pretty well by git in merging.